docs/conf.py

```diff
@@ -151,15 +152,51 @@ def setup(app):
     shutil.copy('../README.md', 'mainREADME.md')
     swap_prefix('mainREADME.md', '```mermaid', '```{mermaid}\n:align: center\n')   
 
+    # Grab the reference file from OR
     url = 'https://raw.githubusercontent.com/The-OpenROAD-Project/OpenROAD/master/docs/contrib/GitGuide.md'
     get_file_from_url(url, 'contrib/GitGuide.md') 
+    
+    # Temporarily using commit number, will change once OR commit merged.
+    url = 'https://raw.githubusercontent.com/The-OpenROAD-Project/OpenROAD/3563176d00daeb613eed2ccf9442137b0ae2cad3/docs/index.md'
```

Comment:
@luarss, what is the PR we are waiting here?

Reply:
https://github.com/The-OpenROAD-Project/OpenROAD/pull/4159

Reply:
@vvbandeira This can be changed after the companion OR PR is merged